src/orange/univ/detail/Types.hh

```diff
@@ -26,12 +26,9 @@ namespace detail
  * bit field to compress the ID and sense at the cost of reducing the max
  * number of surfaces/faces by a factor of two.
  */
-template<class ValueT>
+template<class IdT>
```

Comment:
Isn't that a change in functionality?  (i.e. is it called out in the commit logs).   Namely to obtain the same effective type as before one needs to use `OnTface<OpaqueId< old_template_param > ` ?

Reply:
It's a change to a detail class, only used by the type aliases lower in that file.

Reply:
Exactly why a note in the commit log is appropriate.  The main change (` Open
Use trailing underscore for "sentinel" classes in OpaqueId`) is essential a non functional change.   If we come back later to look at the resulting commit and there is no note, the viewer will 'waste' time understanding how this part of the change is related (especially since it is a (internal but) functional change.